composer.json

```diff
@@ -1,6 +1,48 @@
 {
-	"require": {
-		"phpunit/phpunit": "3.7.24 - 4",
-		"phing/phing": "dev-master"
-	}
+  "name": "koseven/koseven",
+  "description": "Koseven Framework - PHP-7 compatible replacement for Kohana",
+  "license": "BSD",
+  "keywords": [
+    "kohana",
+    "kohana-php7",
+    "koseven"
+  ],
+  "homepage": "https://koseven.ga/",
+  "authors": [
+    {
+      "name": "Koseven Team",
+      "homepage": "https://koseven.ga/",
+      "role": "developer"
+    },
+    {
+      "name": "Kohana Team",
+      "email": "team@kohanaframework.org",
+      "homepage": "http://kohanaframework.org/team",
+      "role": "developer"
+    }
+  ],
+  "autoload": {
+    "psr-0": {
+      "": [
+        "system/classes"
```

Comment:
I added only system based on my usage locally. Others can be added as needed to project composer.json files.